src/js/components/Search.jsx

```diff
@@ -149,24 +150,51 @@ class Search extends Component {
             />
           </div>
           {(() => {
-            if (searchResults.data && searchResults.data.length > 0) {
+            if (!searchResults) {
+              return null
+            }
+        
+            if (searchResults.isFetching) {
               return (
-                <div className={size}>
-                  <Tabs>
-                    <Tab label='Data'>
-                      <div style={{height: '350px'}}>
-                        <DataTable
-                          columns={this.state.columns}
-                          data={searchResults.data}
-                          enableColResize='true'
-                          enableSorting='true'
-                        />
-                      </div>
-                    </Tab>
-                  </Tabs>
-                </div>
+                <span style={{
```

Comment:
We should probably break out this into  a component. We use the loading indicator in a few places already (including safe-framework, so the loading component should probably be put there) and would be useful to just make a component